src/lib/types/article.ts

```diff
@@ -5,19 +5,29 @@ const CategorySchema = z.object({
 	name: z.string()
 });
 
+const AuthorSchema = z.object({
+	username: z.string(),
+	id: z.string(),
+	full_name: z.string(),
+	twitter_username: z.string().nullable()
+});
+
+const RelatedArticleSchema = z.object({
```

Comment:
I'd make more sense if this schema is called BaseArticle, and the one below called FullArticle